scripts/e2e/bootstrap_job/Makefile

```diff
@@ -6,12 +6,16 @@ REGISTRY ?=gcr.io/cluster-api-provider-vsphere/pr/ci
 all: build
 .PHONY : all
 
+.PHONY: genspec
+genspec:
+	../generate-e2e-spec.sh
+
 .PHONY : copyspec
 copyspec:
```

Comment:
this can be deleted

Reply:
We need `genspec`. It generates the spec files using the `manifests` docker image. Did you mean to say we don't need `copyspec`?

Reply:
Yes, the comment is on cooyspec line